### ISSUE TITLE
Filament select befor print dialog box not working

### ISSUE THREAD

**Issue by dkmst** [malnvenshorn/OctoPrint-FilamentManager#91]
I have OctoPi 1.3.7rc3 installed along with filament manager where I have ticked on "Show dialog to confirm selected spools before starting/resuming the print job"

but I am not able or get the option til click "start" is it diable no mater what I do of selection of filament

**Comment by FriendlyGithubBot** [comment]
Hi @dkmst,

It looks like there is some **information missing** in your ticket that will be needed in order to reproduce and fix the problem at hand. Please take a look at the [Issue Template](https://github.com/malnvenshorn/OctoPrint-FilamentManager/blob/master/.github/issue_template.md), which will tell you **exactly** what your ticket has to contain in order to be processable.

If you did not intend to report a bug but have a **question** or want to **request a feature** add a "[Question]" respectively "[Request]" prefix to the title.

I'm marking this one now as an incomplete ticket. Please understand that if you do not provide that information within the next two weeks (until 2018-04-18 00:45) I'll close this ticket so it doesn't clutter the bug tracker.

Best regards,
~ Your FriendlyGithubBot

PS: I'm just an automated script, not a human being.

**Comment by Menschomat** [comment]
Same here! Also error's while choosing the spool in sidebar.

**Comment by Menschomat** [comment]
2018-04-10 12:02:44,732 - octoprint.plugins.filamentmanager - ERROR - Failed to update selected spool for tool0: (psycopg2.ProgrammingError) FEHLER:  Syntaxfehler bei »ON«
LINE 1: ...ES (0, '7ea3ce62-e05b-11e7-83a9-00fd45fdd2b8', 6) ON CONFLIC...
                                                             ^
 [SQL: 'INSERT INTO selections (tool, client_id, spool_id) VALUES (%(tool)s, %(client_id)s, %(spool_id)s) ON CONFLICT ON CONSTRAINT selections_pkey DO UPDATE SET spool_id = %(param_1)s'] [parameters: {'tool': 0, 'param_1': 6, 'client_id': '7ea3ce62-e05b-11e7-83a9-00fd45fdd2b8', 'spool_id': 6}]
2018-04-10 12:02:44,733 - tornado.access - ERROR - 500 PATCH /plugin/filamentmanager/selections/0 (192.168.0.28) 5.51ms
2018-04-10 12:02:45,686 - tornado.access - WARNING - 409 POST /api/printer/command (192.168.0.28) 3.99ms
2018-04-10 12:04:49,922 - octoprint.plugins.filamentmanager - ERROR - Failed to update selected spool for tool0: (psycopg2.ProgrammingError) FEHLER:  Syntaxfehler bei »ON«
LINE 1: ...ES (0, '7ea3ce62-e05b-11e7-83a9-00fd45fdd2b8', 4) ON CONFLIC...
                                                             ^
 [SQL: 'INSERT INTO selections (tool, client_id, spool_id) VALUES (%(tool)s, %(client_id)s, %(spool_id)s) ON CONFLICT ON CONSTRAINT selections_pkey DO UPDATE SET spool_id = %(param_1)s'] [parameters: {'tool': 0, 'param_1': 4, 'client_id': '7ea3ce62-e05b-11e7-83a9-00fd45fdd2b8', 'spool_id': 4}]
2018-04-10 12:04:49,924 - tornado.access - ERROR - 500 PATCH /plugin/filamentmanager/selections/0 (192.168.0.28) 10.63ms

**Comment by malnvenshorn** [comment]
Take a look at #96, seems to be the same issue for me.

**Comment by Menschomat** [comment]
Thanks @malnvenshorn  👍 Now working with Postgre 9.6. You should mark the required Version in Readme.md.

**Comment by Menschomat** [comment]
@dkmst I hope, this will solve your problem, too? I'm running 1.37-Final

**Comment by malnvenshorn** [comment]
> You should mark the required Version in Readme.md.

Already done :wink:

**Comment by dkmst** [comment]
@Menschomat i am on 1.3.7rc3 - but will update today...

So what was the solution to the problem - I dont understand half of what there is written in #96

**Comment by malnvenshorn** [comment]
@dkmst If you would provide a complete ticket with all requested information I might be able to help.

**Comment by malnvenshorn** [comment]
Required information still not provided. Closing.

**Comment by dkmst** [comment]
@Menschomat sry for the delay, i have had a busy time. i will the rest of the things ready to day for you
